### ISSUE TITLE
Custom properties are marked as errors

### ISSUE THREAD

**Issue by dhuebner** [kieler/elkgraph-web#10]
Try 
```
node baseUri {
	layout [ size: 114, 42 ]
	text: "My custom text"
	nodeType: RECTANGLE
}
```

**Comment by uruuru** [comment]
Quite late to reply but be that as it may: It's something that would have to be incorporated into the `elkt` xtext grammar, I assume. 
@spoenemann, @le-cds is this something we ever considered?

**Comment by spoenemann** [comment]
If I remove either `text` or `nodeType` I get the error

> No layout option with identifier 'text/nodeType' can be found.

The grammar allows any property, but unknown properties are marked with errors. We could consider turning those into warnings instead.

When both unknown properties are present, there is another problem: both are resolved to `null`, which leads to a collision of map keys and thus an EMF error message. We should create dynamic properties to avoid that.

**Comment by uruuru** [comment]
> The grammar allows any property, but unknown properties are marked with errors. We could consider turning those into warnings instead

That's kind of what I meant. But I must admit, that I was imprecise :). 

I created eclipse/elk#556 and eclipse/elk#557.

**Comment by uruuru** [comment]
Should be fixed starting with the next ELK release (0.7.0).
